src/pages/Login/Login.layout.vue

```diff
@@ -55,21 +55,44 @@
         </div>
       </q-form>
       <div class="login__other_links">
-        <a class="login__forgotten_link" href="mailto:adidperu@gmail.com"
-          >¿Olvidaste tu contraseña?</a
-        >
-        <a class="login__get_account" href="mailto:adidperu@gmail.com">
+        <button @click="openRecoverPasswordModal" class="login__forgotten_link">
+          ¿Olvidaste tu contraseña?
+        </button>
+        <button @click="openRecoverPasswordModal" class="login__get_account">
           Crear cuenta
-        </a>
+        </button>
       </div>
     </main>
   </div>
 </template>
 
 <script setup lang="ts">
 import didLogo from 'src/assets/logos/didperu.svg';
-import { QInputProps } from 'quasar';
+import { QInputProps, copyToClipboard, Dialog, Notify } from 'quasar';
 
+const openRecoverPasswordModal = () => {
+  Dialog.create({
+    title: 'Recuperar contrasenha',
```

Comment:
```suggestion
    title: 'Recuperar contraseña',
```

Reply:
We are going to use an icon instead of title